ext/hal/st/stm32cube/stm32f1xx/drivers/include/stm32f1xx_hal_conf.h

```diff
@@ -61,13 +61,12 @@
 #define HAL_DAC_MODULE_ENABLED
 #define HAL_DMA_MODULE_ENABLED
 #define HAL_ETH_MODULE_ENABLED
-#define HAL_FLASH_MODULE_ENABLED
-#define HAL_GPIO_MODULE_ENABLED
-#define HAL_HCD_MODULE_ENABLED
-#define HAL_I2C_MODULE_ENABLED

```

Comment:
Please make change more explicit (to avoid confusion on cube update, and change gets erased):
/* #define HAL_I2C_MODULE_ENABLED */ with exclicit comment: what was the exact issue:
"when HAL I2C activated,  I2C_XXX defined in HAL and Zephyr")